main.py

```diff
@@ -1,10 +1,43 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for player_name, player_data in data.items():
+        self_race, created = Race.objects.get_or_create(
+            name=player_data["race"]["name"],
+            description=player_data["race"]["description"],
+        )
+        skills_data = player_data["race"]["skills"]
+
+        for skill_data in skills_data:
+            Skill.objects.get_or_create(
+                name=skill_data["name"],
+                bonus=skill_data["bonus"],
+                race=self_race
+            )
+
+        if player_data["guild"] is not None:
+            self_guild, created = Guild.objects.get_or_create(
```

Comment:
You do not use the `created` variable throughout your code, replace it with `_` here and on line 13.